### PR TITLE
RavenDB-12674 Renew License

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

**Review by grisha-kotler — APPROVED** [review]

**Review by ppekrol — APPROVED** [review]

File: src/Raven.Studio/typescript/viewmodels/shell/registration.ts [hunk 1]
```diff
@@ -182,6 +226,45 @@ class registration extends dialogViewModelBase {
...
+                            !this.renewWhenNotExpired() ) {
+                                app.closeDialog(this);
+                        }
+                        license.fetchSupportCoverage();
+                    });
+                
+                if (result.Error) {
+                    this.renewError(result.Error);
+                    this.renewMessage(null);
+                }
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/shell/registration.ts]: format code:
} else {

File: src/Raven.Studio/wwwroot/Content/css/modals.less [hunk 2]
```diff
@@ -224,6 +224,12 @@
 }
 
 #licenseModal {
+    .licenseModalImg {
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/modals.less:227]: why do we need this?

File: src/Raven.Studio/typescript/viewmodels/shell/registration.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/shell/registration.ts]: done

File: src/Raven.Studio/wwwroot/Content/css/modals.less (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/Content/css/modals.less:227]: well - we don't have to - I just thought the image looks nicer in the dialog when it is a bit **smaller**

**Review by ml054 — APPROVED** [review]

**Pull request merged**